packages/smithy-client/src/parse-utils.spec.ts

```diff
@@ -93,6 +92,28 @@ describe("expectNumber", () => {
   });
 });
 
+describe("expectInt", () => {
+  it("accepts integers", () => {
+    expect(expectInt(1)).toEqual(1);
+  });
+
+  it("accepts null/undefined", () => {
+    expect(expectInt(null)).toEqual(undefined);
+    expect(expectInt(undefined)).toEqual(undefined);
+  });
```

Comment:
Use [jest-each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) for more descriptive tests here and in other tests.
```suggestion
  it.each([null, undefined])("accepts %s", (value) => {
    expect(expectInt(value)).toEqual(undefined);
  });
```